modules/oci-ocne-autoscaling/main.tf

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2023 Oracle Corporation and/or affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+resource "oci_load_balancer_load_balancer" "lb" {
```

Comment:
please rename your resources according to their purpose e.g. is this LB the API or a service load balancer